### ISSUE TITLE
Support for darwin_ARM64 (Apple Macbook Pro with M1 chip)

### ISSUE THREAD

**Issue by saragluna** [aztfmod/terraform-provider-azurecaf#126]
I'm still getting this error when running `terraform init` on my M1 mbp even though it seems like it's been resolved in https://github.com/aztfmod/terraform-provider-azurecaf/issues/95:

```
│ Error: Incompatible provider version
│ 
│ Provider registry.terraform.io/aztfmod/azurecaf v1.2.6 does not have a package available for your current platform, darwin_arm64.
│ 
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms. Other versions of this provider may
│ have different platforms supported.
```

**Comment by junoteam** [comment]
Can you please to add support of darwin_ARM64 (Apple Macbook Pro with M1 chip)?

**Comment by arnaudlh** [comment]
Reopening

**Comment by iemejia** [comment]
I cloned this and it builds perfectly in a M1 however when terraform tries to resolve the provider it seems the arch specific version is not published.
I noticed also that the releases page does not have "terraform-provider-azurecaf_1.2.8_darwin_arm64.zip"
Maybe it is a configuration issue on goreleaser action?

**Comment by arnaudlh** [comment]
Closing as resolved in 1.2.9
